docs/standard/datetime/working-with-calendars.md

```diff
@@ -230,12 +230,13 @@ If relaxed range checks are undesirable, you can restore strict range checks in
    ```
 
 - **.NET Framework 4.5.2 or earlier:** You can set the following registry value:
-|  |  |
-|--|--|
-|Key | HKEY_LOCAL_MACHINE\Software\Microsoft.NETFramework\AppContext |
-|Name | Switch.System.Globalization.EnforceJapaneseEraYearRanges |
-|Type | REG_SZ |
-|Value | 1 |
+
+   |  |  |
```

Comment:
Markdig may complain about this, but the 3 tables are not aligned to the start of the content for the bullet point. That would be two spaces to do that, not 3.